string object_name
---
bool success
string message
